### PR TITLE
Bring back oc10 app release

### PR DESCRIPTION
**Pull request by LukasHirt**
Add build steps to makefile to create a `phoenix-app.tar.gz`, create appinf, change icons to reflect the ones in app switcher and add docs on how to deploy the app.

### CODE REVIEW

File: docs/deployments/oc10-app.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+## Prerequisites
+- Running [ownCloud 10 server](https://owncloud.com/download-server/) with version 10.6
+- Installed [oauth2 app](https://marketplace.owncloud.com/apps/oauth2)
+
+## Deploying ownCloud Web
+To download the ownCloud Web, go to https://github.com/owncloud/phoenix/releases and pick your desired version. In the release, you will find a file called `phoenix-app.tar.gz`.
+Download this file and move it into the `apps` folder in your ownCloud 10 server.
+
+{{< hint info >}}
+The app bundle is provided only from version 1.0.0
```
Comment (LukasHirt) [inline docs/deployments/oc10-app.md:23]: If we decide against releasing version `1.0.0` together with this I'll change this :wink:

File: appinfo/info.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	<summary>Next generation user interface for ownCloud</summary>
+	<description>
+    Next generation user interface for ownCloud based on vue.js.
+    This is an early release with known limitations and missing features.
+
+    Please, report any findings in https://github.com/owncloud/phoenix/issues
+  </description>
+	<licence>AGPLv3</licence>
+	<author>ownCloud</author>
+	<version>1.0.0</version>
```
Comment (LukasHirt) [inline appinfo/info.xml:14]: If we decide against releasing version `1.0.0` together with this I'll change this :wink:

File: docs/deployments/oc10-app.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Next step is to enable the app:
+```bash
+cd ../ && occ apps:enable phoenix
+```
+
+## Configure oauth2
+In the `Admin` of ownCloud 10, head into `User Authentication` and add a new client with arbitrary name and redirection URL `https://<your-owncloud-server>/apps/phoenix/oidc-callback.html`.
+
+## Configure ownCloud 10
+To display ownCloud Web in the app switcher and to redirect all private and public links to the new WebUI, add please following config into `config/config.php`:
```
Comment (phil-davis) [inline docs/deployments/oc10-app.md:40]:
```suggestion
To display ownCloud Web in the app switcher and to redirect all private and public links to the new WebUI, add the following config into `config/config.php`:
```

File: docs/deployments/oc10-app.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+```php
+'phoenix.baseUrl' => 'https://<your-owncloud-server>/apps/phoenix',
+```
+
+## Configure ownCloud Web
+There is a few config values which need to be set in order for ownCloud Web to work correctly. Navigate into `apps/phoenix` and adjust `config.json` according to the following example:
```
Comment (phil-davis) [inline docs/deployments/oc10-app.md:47]:
```suggestion
There are a few config values which need to be set in order for ownCloud Web to work correctly. Navigate into `apps/phoenix` and adjust `config.json` according to the following example:
```

File: docs/deployments/oc10-app.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    "files"
+  ],
+  "applications" : [] // Apps to be displayed in the application switcher
+}
+```
+
+## Accessing ownCloud Web
+After following all the steps, you should see a new entry in the application switcher called `New Design` which points to the ownCloud web.
+
+![image](https://user-images.githubusercontent.com/25989331/101364294-b061f800-38a2-11eb-957e-6115fa7b797b.png)
```
Comment (phil-davis) [inline docs/deployments/oc10-app.md:68]: should this `png` file be committed in the repo, instead of being an external resource?

Comment (LukasHirt) [reply docs/deployments/oc10-app.md]: Sounds good! Will change that

**Review by pmaier1 — APPROVED** [review]
Nice, thanks :+1:

File: docs/deployments/oc10-app.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+{{< toc >}}
+
+One of the build artifacts of the ownCloud Web is a ownCloud 10 app bundle.
+
+## Prerequisites
+- Running [ownCloud 10 server](https://owncloud.com/download-server/) with version 10.6
+- Installed [oauth2 app](https://marketplace.owncloud.com/apps/oauth2)
+
+## Deploying ownCloud Web
+To download the ownCloud Web, go to https://github.com/owncloud/phoenix/releases and pick your desired version. In the release, you will find a file called `phoenix-app.tar.gz`.
```
Comment (LukasHirt) [inline docs/deployments/oc10-app.md:19]: As soon as it's deployed to the marketplace I'll adjust this part to showcase that.

File: Makefile.release [hunk 7]
```diff
@@ -43,6 +56,29 @@ create-release-folder:
 	rm -rf $(CURDIR)/release
 	mkdir $(CURDIR)/release
 
-.PHONY: create-package
-create-package:
+.PHONY: create-packages
```
Comment (micbar) [inline Makefile.release:59]: needs plural in line 52

File: Makefile.release [hunk 8]
```diff
@@ -43,6 +56,29 @@ create-release-folder:
...
+craete-packages: package-plain package-ocx
```
Comment (micbar) [inline Makefile.release:60]: typo `craete/create`

**Review by micbar — CHANGES_REQUESTED** [review]
See comments

File: Makefile.release (hunk 7 above)
Comment (LukasHirt) [reply Makefile.release:59]: Done

File: Makefile.release (hunk 8 above)
Comment (LukasHirt) [reply Makefile.release]: Fixed

**Comment by LukasHirt** [comment]
@micbar Ready for re-review. Both comments addressed

**Comment by micbar** [comment]
tested it. Seems to work. We will test the details in qa.

**Review by micbar — APPROVED** [review]
Nice! Super Fast progress!

**Comment by LukasHirt** [comment]
Had twice in a row failures in CI. I tried to rebase the PR, maybe that'll help

**Pull request merged**